src/overlays/actors/ovl_Bg_Open_Spot/z_bg_open_spot.c

```diff
@@ -27,25 +28,50 @@ const ActorInit Bg_Open_Spot_InitVars = {
     (ActorFunc)NULL,
 };
 
-// static InitChainEntry sInitChain[] = {
 static InitChainEntry D_80ACB3B0[] = {
     ICHAIN_F32(uncullZoneForward, 4000, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneScale, 560, ICHAIN_CONTINUE),
     ICHAIN_F32(uncullZoneDownward, 800, ICHAIN_CONTINUE),
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-#endif
+void BgOpenSpot_Init(Actor* thisx, GlobalContext* globalCtx) {
+    BgOpenSpot* this = THIS;
 
-extern InitChainEntry D_80ACB3B0[];
+    Actor_ProcessInitChain(&this->actor, D_80ACB3B0);
+    this->animatedMaterial = Lib_SegmentedToVirtual(object_open_obj_Matanimheader_002CE0);
```

Comment:
You could name this and the textures it uses, it's just a minor thing that would finish this up, rather than someone document the whole object at once.

Reply:
I'll give it a shot :)

Reply:
I hope I've taken the right approach to this.